dockerfiles/hpcc/templates/_util.tpl

```diff
@@ -90,3 +90,22 @@ volumeMounts:
 image: "{{ include "hpcc.utils.imageName" . }}"
 imagePullPolicy: {{ .root.Values.global.image.pullPolicy }}
 {{- end -}}
+
```

Comment:
should change eclccserver.yaml to use this too

Reply:
true, yes, I meant to.. will change